### DISCUSSION TITLE
cw not work

### DISCUSSION THREAD

**Discussion by tham9500** [aws/codewhisperer-command-line-discussions#109] [Known bugs]
cw not work

**Comment by jacklvd** [comment]
I also have that issues when change from fig to cw, the cw version is 0.15.0

**Comment by Metal-spoon** [comment]
Hi, I faced a similar issue when i went from fig to cw.
Try the following:
`cd /Applications/CodeWhisperer.app/Contents/MacOS` 

Then run: 
`./cw doctor`

If my hunch is correct it will ask you to login, do so by running: 
`./cw login`

now run `./cw doctor` again, if my hunch is correct it should tell you that `~/.local/bin` is missing from your path.

To fix this run:
`./cw integrations install dotfiles`

Restart your shell and run `cw doctor` **note, without the `./` because it should be added to your PATH variable now**
Installing codewhisperer somehow doesn't add the binary to your path, seems to be an oversight.

**Reply by manojchandrashekar** [reply]
Thank you! This solves the issue.

**Reply by jacklvd** [reply]
Thank you this resolves the issue.

**Reply by adnjoo** [reply]
thanks!

**Reply by grant0417** [reply]
You should be able to launch the app and it will handle these for you.

**Reply by grant0417** [reply]
Okay, figured out the issue here, the dashboard was having an issue where some of the functionality for login was not working. This should be resolved if you update and restart the app.
